orm/installation.md

```diff
@@ -27,16 +30,17 @@ Optionally you can register the EntityManager, Registry and/or Doctrine facade:
 'Doctrine'      => LaravelDoctrine\ORM\Facades\Doctrine::class,
 ```
 
+## Config
 To publish the config use:
 
 ```php
-php artisan vendor:publish --tag="config"
+php artisan vendor:publish --tag="config" --provider="LaravelDoctrine\ORM\DoctrineServiceProvider"
 ```
 
 Available environment variables inside the config are: `APP_DEBUG`, `DOCTRINE_METADATA`, `DB_CONNECTION`, `DOCTRINE_PROXY_AUTOGENERATE`, `DOCTRINE_LOGGER` and `DOCTRINE_CACHE`
 
 > Important:
-> By default, Laravel's application skeleton has its `Model` classes in the `app/` folder. With Doctrine, you'll need to
+> By default, Laravel's application skeleton has its `Model` classes in the `app/Models` folder. With Doctrine, you'll need to
```

Comment:
Should this be Entities now?

Reply:
no this is laravel `Models` but add "we suggest `app/Entities`"  below